src/views/badge.j2

```diff
@@ -0,0 +1,8 @@
+<p><b>Badge #:</b> {{ Serial }}</p>
+<p><b>Name:</b> {{ name }}</p>
+<p><b>Rank:</b> {{ RankRole }}</p>
+{% if job_title %}
+<p><b>Job Title:</b> {{ job_title }}</p>
+<p><b>Hourly Rate:</b> ${{ hourly_rate }}</p>
+<p><b>Projected salary (w/o overtime):</b> ${{ projected }}</p>
+{% endif %}
```

Comment:
Ah I was just gonna ask about Jinja2 templates 👌

Reply:
Yea I wanted to get something out the door yesterday so that's why I just said screw it and shipped, but this is a much better approach!